protos/IncomingPushMessageSignal.proto

```diff
@@ -103,6 +77,34 @@ message DataMessage {
   optional uint32             flags       = 4;
   optional uint32             expireTimer = 5;
   optional bytes              profileKey  = 6;
+  optional uint64             timestamp   = 7;
+}
+
+message NullMessage {
+  optional bytes padding = 1;
+}
+
+message ReceiptMessage {
+  enum Type {
+    DELIVERY = 0;
+    READ     = 1;
+  }
+
+  optional Type   type      = 1;
+  repeated uint64 timestamp = 2;
```

Comment:
Looks like this is a change that will need to be reflected in the code (`timestamps` -> `timestamp`)

Reply:
Great catch 🎣 👏

Reply:
Made the change: 359e7e482f8f09fee2fe55cb97017126ffc932ac Will test it more thoroughly tomorrow.

Reply:
Tested this now in both groups (2 desktop + 1 mobile) and private messages with read receipts enabled and disabled.